src/components/Datepicker/helpers.ts

```diff
@@ -119,8 +119,9 @@ export const getFormattedDate = (
 
   if (dateFormat) {
     return DateFNSFormat(new Intl.DateTimeFormat(language, defaultOptions).format(date), dateFormat);
+  } else {
```

Comment:
Can't we just remove the `DateFNSFormat` for everything?

Reply:
```ts
 if (dateFormat) {
    return DateFNSFormat(date), dateFormat);
  } else {
    return new Intl.DateTimeFormat(language, defaultOptions).format(date);
  }
```

@rluders - you meant something like this?

Reply:
@rluders `dateFormat` will have values like `dd-MMM-yyyy` or `dd MMM yyyy` or `dd-mm-yyyy` to format the date Date appearences.

`dd-MMM-yyyy` // output => 11-Mar-2024
`dd MMM yyyy` // output => 11 Mar 2024
`dd-mm-yyyyy` // output => 11-03-2024


I am having `if` condition because if the users aren't passing the `inputFormat` props which I am calling as `dateFormat` in `getFormattedDate()` function, then else block be executed.

Sorry if I've mistaken or confused anywhere.

Reply:
@rluders -- your feedback, please

Reply:
## dd-MMM-yyyy // output => 11-Mar-2024
![image](https://github.com/themesberg/flowbite-react/assets/32231977/bd927495-95f5-4a5b-9c36-8f9d804aacc8)

## dd MMM yyyy // output => 11 Mar 2024
![image](https://github.com/themesberg/flowbite-react/assets/32231977/869825b9-51c6-42f6-b83f-b223cdf527f2)

## dd-MM-yyyyy // output => 11-03-2024
![image](https://github.com/themesberg/flowbite-react/assets/32231977/56df6a33-d858-4fd9-b3b1-72a04eab70ab)


@rluders -- for the Date Format, I meant with this, like many times User needs to show/display the date in the required/custom format, this PR solves that issue.

Reply:
I'll check it ASAP.

Reply:
@rluders OK Sir.... Sorry for the followup 🙏